docs/topic-guides/tls.txt

```diff
@@ -0,0 +1,68 @@
+.. _tls:
+
+##########################
+ Transport Layer Security
+##########################
+
+**Transport layer security** (TLS) is a protocol for applying encryption
+to network communications. It prevents the data being transmitted from
```

Comment:
Technically, the first two sentences aren't the whole truth: TLS is about providing secure network communications, which includes not just preventing messages from being observed, but also providing sender/receive identity and preventing messages from being modified in transit.